docs/experiments/taint-propagators.md

```diff
@@ -49,14 +49,13 @@ pattern-propagators:
 ```
 
 In the example above, whenever Semgrep finds the pattern `$X.add($Y)`, it checks if
-`$Y` is tainted. If `$Y` is tainted, Semgrep propagates that taint to `$X`. Thus,
-adding tainted data to a set marks the set itself as tainted.
+`$Y` is tainted. If `$Y` is tainted, Semgrep propagates that taint to `$X`. 
+Thus, adding tainted data to a set marks the set itself as tainted.
 
 <iframe src="https://semgrep.dev/embed/editor?snippet=7lNe" border="0" frameBorder="0" width="100%" height="432"></iframe>
 
-## Propagator rule requirements
+## Requirements of taint propagator rules
 
-A propagator rule necessitates:
+A taint propagator rules necessitates:
 - Any arbitrary pattern that must bind two metavariables of your choice.
-- You must specify the `from` and the `to` metavariables.
-
+- You must specify the `from` and the `to` metavariables. TODO
```

Comment:
@IagoAbal can we say something about `from` and the `to` metavariables? Is this specifying the "span" of variables that Semgrep should look for?

Reply:
Some details on [page 5 of my script](https://docs.google.com/document/d/1hqB-a_wkqrEwk_ZsFnlZKOFo5HPeUdsGPMRj8umTDxk/edit). 
```suggestion
- You must specify the `from` and the `to` metavariables. TODO
-  The `from` and `to` metavariables are required. `from` indicates where the tainted source must to pass “through” to propagate the taint. `to` specified what additional object or structure is tainted as a result.
```
@adamkvitek I'm sure you can clean up that suggestion to fit the tech writer standards, but that's a first stab.